src/sdk/pynni/nni/nas_utils.py

```diff
@@ -265,6 +265,8 @@ def convert_nas_search_space(search_space):
         param search_space: raw search space
         return: the new search space, mutable_layers will be converted into choice
     """
+    if not isinstance(search_space, dict):
+        return search_space
```

Comment:
in what scenario that search_space is not a dict?

Reply:
Users might write a customized search space, which is a list. Better careful than sorry. This won't be harmful.